toolkit/tools/imager/imager.go

```diff
@@ -621,17 +625,17 @@ func buildImage(mountPointMap, mountPointToFsTypeMap, mountPointToMountArgsMap,
 		var initramfsPathList []string
 		err = readOnlyRoot.SwitchDeviceToReadOnly(mountPointMap["/"], mountPointToMountArgsMap["/"])
 		if err != nil {
-			err = fmt.Errorf("failed to switch root to read-only: %w", err)
+			err = fmt.Errorf("failed to switch root to read-only:\n%w", err)
 			return
 		}
 		installutils.ReportAction("Hashing root for read-only with dm-verity, this may take a long time if error correction is enabled")
 		initramfsPathList, err = filepath.Glob(filepath.Join(installRoot, "/boot/initrd.img*"))
 		if err != nil || len(initramfsPathList) != 1 {
-			return fmt.Errorf("could not find single initramfs (%v): %w", initramfsPathList, err)
+			return fmt.Errorf("could not find single initramfs (%v):\n%w", initramfsPathList, err)
```

Comment:
direct return here instead of assigning the result of `fmt.Errorf` to `err`

Reply:
Trying to minimise the changes, I think both are ok